src/components/CookiesBanner.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {FunctionComponent, useEffect, useState} from "react";
+import {Cookies} from "react-cookie";
+import CookieBanner from 'react-cookie-banner';
+import styles from "./CookiesBanner.module.css";
+
+export const CookiesMessageBanner: FunctionComponent = () => {
+	const cookies = new Cookies();
+	const [init, setInit] = useState<boolean>(false);
+
+	useEffect(() => {
+		setInit(true);
+	}, []);
+
+	if (!init || !cookies || cookies.get("accepts-cookies")) {
```

Comment:
Looks like the lines are not needed - I checked both in dev and prod build:
```suggestion
	if (!cookies || cookies.get("accepts-cookies")) {
```